libraries/helpers.rb

```diff
@@ -16,7 +16,7 @@ def arch_64?
     end
 
     def windows?
-      node['platform_family'].eql?('windows') ? true : false
+      node.platform_family?('windows') ? true : false
```

Comment:
Isn't the definition of a function that ends in question mark that it's a predicate? And doesn't that mean it returns true or false already?